### ISSUE TITLE
Document MongoDB production settings

### ISSUE THREAD

**Issue by rglover** [cheatcode/nodejs-server-boilerplate#13]
How you configure these in production depends on how your MongoDB is hosted. The current settings will work fine with an externally hosted MongoDB instance, but not with one that is run alongside the app (e.g., as a service on the host machine).

**Comment by inkiltie** [comment]
I just created `settings-production.json` and added mongo url of newly created free Atlas sandbox:

```
  "databases": {
    "mongodb": {
      "uri": "mongodb+srv://user:password@cluster0.r1kak.mongodb.net/collection?retryWrites=true&w=majority"
    }
  },
```

Everything works fine.

**Comment by pisacode** [comment]
I am getting ` MongoNetworkError: connection <monitor> to 25.123.121.34:27017 closed` error whenever I tried to connect to Atlas MongoDB which has no IP restrictions.

**Comment by sumit12690** [comment]
How app will connect with mongodb in production? you are performing mongodb connection in .app/reset.js but that file is not included when we are making production build.

**Comment by sumit12690** [comment]
Can we attach mysql and postgreSQL also in same repo so that if someone will have multiple DB then also he can use this as boilerplate?

**Comment by rglover** [comment]
@sumit12690 yes, it's possible, just need to wire up the starting of the database to the `/.app/development.js` file and then add an adapter similar to the `/lib/mongodb.js` file. I'll be working on this for [Joystick](https://github.com/cheatcode/joystick) (our full-stack framework) in a few weeks and will have some patterns that can be implemented in this boilerplate.

**Comment by sumit12690** [comment]
> @sumit12690 yes, it's possible, just need to wire up the starting of the database to the `/.app/development.js` file and then add an adapter similar to the `/lib/mongodb.js` file. I'll be working on this for [Joystick](https://github.com/cheatcode/joystick) (our full-stack framework) in a few weeks and will have some patterns that can be implemented in this boilerplate.

Sounds Good @rglover. Let me know when you are done with that. Thanks!
